tests/Feature/Regression/Issue1551SettingsRemoveLogoTest.php

```
@@ -0,0 +1,144 @@
+        $this->assertResponseRedirectsToRoute($response, 'settings');
+    }
+
+    #[Test]
+    public function remove_logo_requires_post_and_csrf(): void
+    {
+        /* Act - Try to access remove_logo via GET (should fail) */
+        $getResponse = $this->get('/settings/remove_logo/invoice');
+
+        /* Assert - GET request is rejected */
+        self::assertNotSame(200, $getResponse->statusCode(), 'GET request to remove_logo should fail');
+    }
```

PHPMD (2.15.0)

[error] The method remove_logo_requires_post_and_csrf is not named in camelCase. (undefined)

(CamelCaseMethodName)